geometry/scalar.h

```diff
@@ -5,13 +5,24 @@
 #pragma once
 
 #include <cfloat>
+#include <limits>
+#include <valarray>
 
+#include "flutter/fml/logging.h"
 #include "impeller/geometry/constants.h"
 
 namespace impeller {
 
 using Scalar = float;
 
+constexpr bool ScalarNearlyEqual(
```

Comment:
Nit: Does constexpr imply inline or are you just missing it here? I must admit I never remember and add both.